test/e2e/install_test.go

```diff
@@ -56,6 +58,7 @@ func TestCrossplaneLifecycle(t *testing.T) {
 			WithLabel(LabelSize, LabelSizeSmall).
 			WithLabel(LabelModifyCrossplaneInstallation, LabelModifyCrossplaneInstallationTrue).
 			WithLabel(config.LabelTestSuite, config.TestSuiteDefault).
+			WithLabel(config.LabelTestSuite, TestSuiteLifecycle).
```

Comment:
```suggestion
			WithLabel(config.LabelTestSuite, TestSuiteLifecycle).
```

Reply:
This was intentional, features can have multiple values for each label. This allows the environment config suite to run the upgrade tests and not the whole default suite.